src/test/java/InviteTest.java

```diff
@@ -0,0 +1,209 @@
+                .build();
+
+        litmusClient.authenticate(LoginRequest.builder().username(invitee).password(password).build());
+        CommonResponse response = litmusClient.acceptInvitation(request);
+        assertThat(response.getMessage()).isEqualTo("Successful");
+    }
+
+    @Test
+    public void testDeclineInvitation() throws IOException, LitmusApiException {
+        createInvite(projectId, userId);
+
+        DeclineInvitationRequest request = DeclineInvitationRequest.builder()
+                .projectId(projectId)
+                .userId(userId)
+                .build();
+
+        litmusClient.authenticate(LoginRequest.builder().username(invitee).password(password).build());
+        CommonResponse response = litmusClient.declineInvitation(request);
+        assertThat(response.getMessage()).isEqualTo("Successful");
+    }
+
+    @Test
+    public void testRemoveInvitation() throws IOException, LitmusApiException {
+        String userTestName = generateRandomUsername();
+        user = createPasswordUpdatedTestUser(userTestName, oldPassword, TEST_USER_ROLE, TEST_USER_EMAIL, TEST_USER_NAME, password);
+        userId = user.getUserID();
+        invitee = user.getUsername();
+        createInvite(projectId, userId);
+        RemoveInvitationRequest request = RemoveInvitationRequest.builder()
+                .projectId(projectId)
+                .userId(userId)
+                .build();
+
+        litmusClient.authenticate(LoginRequest.builder().username(username).password(password).build());
+        CommonResponse response = litmusClient.removeInvitation(request);
+        assertThat(response.getMessage()).isEqualTo("Successful");
+    }
+
+    @Test
+    public void testListInvitation() throws IOException, LitmusApiException {
+        String userTestName = generateRandomUsername();
+        user = createPasswordUpdatedTestUser(userTestName, oldPassword, TEST_USER_ROLE, TEST_USER_EMAIL, TEST_USER_NAME, password);
+        userId = user.getUserID();
+        invitee = user.getUsername();
+        createInvite(projectId, userId);
+
+        litmusClient.authenticate(LoginRequest.builder().username(invitee).password(password).build());
+        List<ListInvitationResponse> response = litmusClient.listInvitation("Pending");
+        assertThat(response.size()).isEqualTo(1);
+    }
+
+    @Test
+    public void testInviteUsers() throws IOException, LitmusApiException {
+        litmusClient.authenticate(LoginRequest.builder().username(username).password(password).build());
+
+        List<String> tags = Arrays.asList("tag1", "tag2");
+        String projectName = generateUniqueProjectName("Temporary Project");
+        CreateProjectRequest projectRequest = CreateProjectRequest.builder()
+                .projectName(projectName)
+                .description("testDescription")
+                .tags(tags)
+                .build();
+
+        ProjectResponse projectResponse = litmusClient.createProject(projectRequest);
```

Comment:
I think we need to remove created project after test finished for independent tests.